app/course/services/courseStateService.js

```diff
@@ -429,11 +429,7 @@ courseApp.service('courseStateService', function ($rootScope, $log, Course, Term
 						isCourseDeleteModalOpen: false
 					};
 
-					// lock the table if all terms are locked
-					uiState.tableLocked = action.payload.terms
-						.map(function (term) { return new Term(term); })
-						.every(function (term) { return term.isLocked(); });
-
+					uiState.tableLocked = false;
```

Comment:
Is the 'tableLocked' variable still needed?

Reply:
Table locking is used by the 'course import' UI and when adding new courses.